sass/components/_info-toast.scss

```diff
@@ -0,0 +1,65 @@
+@charset 'UTF-8';
+
+.info-toast {
```

Comment:
Could you actually move this file to live next to the component that it's used in? You can also import it directly into the JS file instead of `_module.scss` so that it's compiled into the same JS bundle as the component. Right now, it's loaded as part of the page load instead of potentially lazy loaded with the rest of the sidebar

Reply:
Updated as suggested, thanks